pkg/runtime/runtime.go

```diff
@@ -2397,7 +2397,7 @@ func (a *DaprRuntime) processComponentSecrets(component components_v1alpha1.Comp
 
 		resp, ok := cache[m.SecretKeyRef.Name]
 		if !ok {
-			r, err := secretStore.GetSecret(secretstores.GetSecretRequest{
+			r, err := secretStore.GetSecret(context.Background(), secretstores.GetSecretRequest{
```

Comment:
Here too. Please use `context.TODO()` and/or add a comment: `TODO: Cascade context`

Reply:
Fixed.